sass/components/_popover.scss

```diff
@@ -27,17 +27,17 @@
     &.channel-header__popover {
         z-index: 999;
         margin-left: -4px;
-        font-size: em(12px);
+        font-size: 12px;
 
         &.chanel-header__popover--dm {
-            margin-left: 70px;
+            margin-left: 69.5px;
```

Comment:
Out of curiosity, why is the `.5px` required? I haven't encountered a situation in webapp that required this before.

Reply:
I think this is a combination of a mix between em's and pixels somewhere down the line. I used those values because this is the most correct adjustment for the popover. We can address this in the ticket above as well.